### PR TITLE
File Uploader improvements

### PR DESCRIPTION
**Pull request by dsamojlenko**
# Description

File Uploader improvements:
- improved styling
- enable remove link

# Test Instructions

- Navigate to add_condition
- Click Upload button and select a file
- Notice the lovely styling
- Try out the remove link

### CODE REVIEW

File: assets/js/components/FileUpload.vue [hunk 1]
```diff
@@ -1,45 +1,51 @@
...
+    removeFile(file) {
+      this.uploaded_files.splice(this.uploaded_files.indexOf(file), 1)
+    },
+  },
+  mounted() {
+    console.log(this.fieldName)
+    if (this.files) {
+      this.uploaded_files = this.files.split(',')
+    }
+    console.log('Component ready.')
```
Comment (jeffmaher) [inline assets/js/components/FileUpload.vue]: Stray `console.log`?

**Review by jeffmaher — DISMISSED** [review]
Works in Firefox! LGTM 🍕 . Left one non-blocking comment.

File: assets/js/components/FileUpload.vue (hunk 1 above)
Comment (dsamojlenko) [reply assets/js/components/FileUpload.vue]: yup, removed ... there was another one just above.

**Review by jeffmaher — DISMISSED** [review]
🍕

**Review by CalvinRodo — CHANGES_REQUESTED** [review]
I'm not getting styling browser is chrome: 

Screen Shot 2019-11-25 at 9 20 31 AM

and also 

The contrast on the text: `These could include: clinical notes, specialist reports, or hospital discharge reports, etc` is not accessible. 

Error text from Axe-Chrome: 

>Element has insufficient color contrast of 1.48 (foreground color: #cbd5e0, background color: #ffffff, font size: 15.6pt (20.8px), font weight: normal). Expected contrast ratio of 4.5:1

Although i notice that text is not part of this PR, but it seems related in scope to this PR so flagging it here.

**Comment by CalvinRodo** [comment]
The contrast on the text: `These could include: clinical notes, specialist reports, or hospital discharge reports, etc` is not accessible. 

Error text from Axe-Chrome: 

>Element has insufficient color contrast of 1.48 (foreground color: #cbd5e0, background color: #ffffff, font size: 15.6pt (20.8px), font weight: normal). Expected contrast ratio of 4.5:1

**Comment by dsamojlenko** [comment]
Fixed the heroku Review App issue here - need to include .vue files when purging css

**Review by jeffmaher — APPROVED** [review]
🍕

**Review by CalvinRodo — APPROVED** [review]

**Pull request merged**